public/help/skins-approval.html

```diff
@@ -17,9 +17,9 @@
   <dt>External images</dt>
   <dd>
     We've changed our policy on this now! Public skins CAN contain images now, but so we can make sure that the image links will not break, we will download the images to our own servers. Please only share your skin if you are okay with our doing this! We do also ask that if possible you
-    attribute your images with comments in the css, like this: 
+    attribute your images with comments in the "description" field, like this: 
```

Comment:
It should actually be capitalized to match the label on the form, i.e. `"Description" field` with the big `D`. Sorry I wasn't clear about that!